### PR TITLE
Styled DropdownFieldMulti

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://trello.com/c/95Nr1LgN/94-migrate-composer-forms-to-styled-components

### CODE REVIEW

File: packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <h1>DropdownField initial option</h1>
+    <DropdownFieldMulti
+      placeholder={'Select an item'}
+      onItemChange={action('On click icon')}
+      options={options}
+      initialSelectedOption={options[2]}
+    />
+  </>
+);
+
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.stories.tsx:39]: missing option with multiple items selected

File: packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.test.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  });
+  it('should render initial option', () => {
+    givenComponentRendered({ initialSelectedOption: options[1] });
+    screen.getByText('Option 2');
+  });
+  it('should render disabled', () => {
+    givenComponentRendered({ disabled: true });
+    const dropdown = screen.getByTestId(testId);
+    expect(dropdown).toBeDisabled();
+  });
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.test.tsx:64]: missing test cases for customCss and className

File: packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/index.tsx [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './DropdownFieldMulti';
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/index.tsx:1]: missing _global_ export

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.test.tsx (hunk 2 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.test.tsx:64]: Added

File: packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/index.tsx (hunk 3 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/index.tsx:1]: Added

File: packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.stories.tsx (hunk 1 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownFieldMulti/DropdownFieldMulti.stories.tsx:51]: Added

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**